code/datums/ammo/bullet/revolver.dm

```diff
@@ -27,8 +27,9 @@
 	penetration = ARMOR_PENETRATION_TIER_4
 	accuracy = HIT_ACCURACY_TIER_3
 
-/datum/ammo/bullet/revolver/heavy/on_hit_mob(mob/M, obj/projectile/P)
-	knockback(M, P, 4)
+/datum/ammo/bullet/revolver/heavy/on_hit_mob(mob/entity, obj/projectile/bullet)
+	slowdown(entity, bullet, 4)
```

Comment:
Slowdown doesn't take 3 args

Reply:
I don't understand this statement.

Reply:
`/datum/ammo/proc/slowdown(mob/living/living_mob, obj/projectile/fired_projectile)`

Argument 1 is living_mob, expected to be a /mob/living; you pass entity to be this argument
Argument 2 is fired_projectile, expected to be a /obj/projectile/; you pass bullet to be this argument
Argument 3 does not exist, but you pass 4 into a non-existent argument.